docs/topics/opt-in-requirements.md

```diff
@@ -145,9 +145,10 @@ If you build your module with Gradle, you can add arguments like this:
 import org.jetbrains.kotlin.gradle.tasks.KotlinCompilationTask
 // ...
 
-val compileKotlin = tasks.named<KotlinCompilationTask<*>>("compileKotlin")
+tasks.named<KotlinCompilationTask<*>>("compileKotlin").configure {
+    compileKotlin.compilerOptions.freeCompilerArgs.add("-opt-in=org.mylibrary.OptInAnnotation")
```

Comment:
Hm, in 1.9.0 release there is a separate option for `optIn`:
`compilerOptions.optIn.add("..")`

@PetrakovichVictoria should we update this PR and wait for `1.9.0` final release or merge it as is and then, after release, update? :thinking: